infra/pr_helper.py

```diff
@@ -223,6 +228,22 @@ def get_pull_request_number(self, commit):
       return None
     return pr_response.json()[0]['number']
 
+  def get_past_contributors(self, project_path):
+    """Returns a list of past contributors of a certain project."""
+    commits_response = requests.get(f'{BASE_URL}/commits?path={project_path}',
+                                    headers=self._headers)
+
+    if commits_response.ok:
+      commits = commits_response.json()
+      contributors = []
+      for commit in commits:
+        if commit['author']['login'] not in contributors:
+          contributors.append(commit['author']['login'])
```

Comment:
Is it possible to tell when these logins are verified?

Reply:
I think the verification information we have is for each commit. However, the login (username) information is found in the "author" field, not the "commit" field. I understand that people may forge email addresses, as they are found in the commit message. But, can people forge the 'author' information? API sample: https://api.github.com/repos/google/oss-fuzz/commits?path=projects/adal

Reply:
I believe the "author" is just mapped based on the provided email which may be forged.

Reply:
Thanks!

Reply:
Sample comment: https://github.com/google/oss-fuzz/pull/10663#issuecomment-1628073057